client/root-entry.js

```diff
@@ -0,0 +1,27 @@
+/**
+ * External dependencies
+ */
+import page from 'page';
+
+/**
+ * Internal dependencies
+ */
+import { abtest } from 'lib/abtest';
+import userFactory from 'lib/user';
+
+export default function() {
+	page( '/', context => {
+		const user = userFactory();
+		const { primarySiteSlug } = user.get();
```

Comment:
We'd like to eventually deprecate `lib/user` and replace it with the Redux store.

In this case, the following Redux code should do the same job:
```
const { primarySiteSlug } = getCurrentUser( context.store.getState() );
```